README.md

```diff
@@ -117,6 +127,8 @@ Tracez le graphique correspondant.
 
 Quel semble être la complexité de notre fonction de tri ? Cela est-il logique par rapport au code que vous avez implémenté ?
 
+- La compléxité semble être de O(N²). Cela paraît logique, mais elle pourrait peut-être être aussi égale à O(2^n) car il y a une boucle while dans une boucle for dans une boucle for
```

Comment:
Il ne faut pas prendre en compte la boucle for qui génère les tableaux de tests : elle ne fait pas partie de notre algorithme de tri